tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -206,3 +268,256 @@ def test_positive_resource_vm_power_management(
+    """"Associate custom default (3-Large) compute profile to RHV compute resource.
+
+    :id: e7698154-62ff-492b-8e56-c5dc70f0c9df
+
+    :customerscenario: true
+
+    :setup: rhev hostname and credentials.
+
+    :steps:
+
+        1. Create a compute resource of type RHV.
+        2. Provide it with the valid hostname, username and password.
+        3. Select the created rhv CR.
+        4. Click Compute Profile tab.
+        5. Edit (3-Large) with valid configurations and submit.
+
+    :expectedresults: The Compute Resource created and associated to compute profile (3-Large)
+        with provided values.
+
+    :BZ: 1286033
+
+    :Caseautomation: Automated
+    """
+    cr_name = gen_string('alpha')
+    cr_profile_data = dict(
+        cluster=rhev_data['datacenter'],
+        cores='2',
+        sockets='2',
+        memory='1 GB',
+        network_interfaces=[
+            dict(name='nic1', network=settings.vlan_networking.bridge),
+            dict(name='nic2', network=settings.vlan_networking.bridge),
+            dict(name='nic3', network=settings.vlan_networking.bridge),
+        ],
+        storage=[
+            dict(
+                size='10',
+                storage_domain=rhev_data['storage_domain'],
+                bootable=False,
+                preallocate_disk=True,
+            ),
+            dict(
+                size='20',
+                storage_domain=rhev_data['storage_domain'],
+                bootable=True,
+                preallocate_disk=True,
+            ),
+            dict(
+                size='5',
+                storage_domain=rhev_data['storage_domain'],
+                bootable=False,
+                preallocate_disk=False,
+            ),
+        ]
+    )
+    with session:
+        session.computeresource.create({
+            'name': cr_name,
+            'provider': FOREMAN_PROVIDERS['rhev'],
+            'provider_content.url': rhev_data['rhev_url'],
+            'provider_content.user': rhev_data['username'],
+            'provider_content.password': rhev_data['password'],
+            'provider_content.datacenter.value': rhev_data['datacenter'],
+            'provider_content.certification_authorities': module_ca_cert,
+        })
+        assert session.computeresource.search(cr_name)[0]['Name'] == cr_name
+        session.computeresource.update_computeprofile(
+            cr_name,
+            COMPUTE_PROFILE_LARGE,
+            {'provider_content.{0}'.format(key): value for key, value in cr_profile_data.items()}
+        )
+        provider_content_values = session.computeresource.read_computeprofile(
+            cr_name, COMPUTE_PROFILE_LARGE)['provider_content']
+        # assert main compute resource profile data updated updated successfully.
+        excluded_keys = ['network_interfaces', 'storage']
+        assert (
+            {key: value for key, value
+             in cr_profile_data.items() if key not in excluded_keys}
+            ==
```

Comment:
non-blocking suggestion: this assertion and the next one are pretty hard to read. Would you mind storing these dicts in some `expected_data`/`actual_data` variables (or smth similar)?

Reply:
implemented